library/bignum.c

```diff
@@ -2210,20 +2210,20 @@ static int mpi_check_small_factors(const mbedtls_mpi *X)
     int ret = 0;
     size_t i;
     mbedtls_mpi_uint r;
+    unsigned p = 3; /* The first odd prime */
```

Comment:
not important: I know this is defined to integer but is it better to state the type just for readability?

Reply:
I think @waleed-elmelegy-arm is talking about `unsigned` vs `unsigned int`

Reply:
We don't have a style rule or habit for writing out `int` after `unsigned`.